lib/pure/hashes.nim

```diff
@@ -121,16 +121,32 @@ proc hash*(x: string): THash =
     h = h !& ord(x[i])
   result = !$h
 
+# The following need a better home than "hashes":
+const magicIdentSeparatorRuneByteWidth* = 3
+
+proc isMagicIdentSeparatorRune*(cs: cstring, i: int): bool  {. inline } =
+  result =  ord(cs[i]) == 226 and 
+            ord(cs[i + 1]) == 128 and
+            ord(cs[i + 2]) == 147     # en-dash  # 145 = nb-hyphen
+
 proc hashIgnoreStyle*(x: string): THash =
```

Comment:
It seems hashIgnoreStyle is referenced from these files:
/compiler/idents.nim
/compiler/magicsys.nim
/compiler/wordrecg.nim
/lib/pure/gentabs.nim
/lib/pure/strtabs.nim

The lib-refs mucks up a move to compiler/...

So, since their used from "lib/pure/hashes.nim" and "compiler/idents.nim", perhaps I should add a "lib/system/priv_etc.nim" (do you have a convention for naming modules that are not 'end-user public' and should not be used outside of compiler and stdlib? 'priv' was just first prefix that came to mind), "etc" for "not easily categorizable / or to small to warrant a specific mod"... ?